### DISCUSSION TITLE
Keep getting errors on reguler arch

### DISCUSSION THREAD

**Discussion by GudatSpeleng** [sherlock-project/sherlock#2256] [Q&A]
When i try the first install under the github tab it gives me the errors "bash: virtualenv: command not found
bash: bin/activate: No such file or directory" if I go past that "pipx install -e ." works fine, but when i do "sherlock user123" 
it gives me the bash error "bash: sherlock: command not found" any help is welcomed

**Comment by GudatSpeleng** [comment]
i tried it again and looks like this fatal error popped up or i just missed it (sorry)
"fatal: destination path 'sherlock' already exists and is not an empty directory." should i go into my .bin and make a folder myself?

**Comment by ppfeister** [comment]
Hey @GudatSpeleng 

Just to confirm, are you hoping to work with the source code itself in some capacity?

virtualenv is the preferred method for a lot of people, but not everyone uses it. I often use the venv module instead, since it's native (`python -m venv .venv`).

Alternatively, since this version uses a poetry backend, you can skip the venv and pip install and just use `poetry install` and `poetry run sherlock xyz` for devel use. Although, this requires Poetry to be installed first.

**Reply by GudatSpeleng** [reply]
i was trying to run the source code because i was having problems with the pipx install, when i tried to run sherlock --version it gave me an error, so i tried to use the source code. i reinstalled the pip way, turns out i forgot to run `pipx ensurepath`  after running `pipx install sherlock-project`.  Thank you for helping me! I appreciate it.
